pkg/security/api/types.go

```diff
@@ -0,0 +1,86 @@
+package api
+
+import (
+	"k8s.io/kubernetes/pkg/api/unversioned"
+	kapi "k8s.io/kubernetes/pkg/api"
+)
+
+// PodSecurityPolicySubjectReview checks whether a particular user/SA tuple can create the PodSpec.
+type PodSecurityPolicySubjectReview struct {
+	unversioned.TypeMeta
+
+	// Spec defines specification for the PodSecurityPolicySubjectReview.
+	Spec PodSecurityPolicySubjectReviewSpec
+
+	// Status represents the current information/status for the PodSecurityPolicySubjectReview.
+	Status PodSecurityPolicySubjectReviewStatus
+}
+
+// PodSecurityPolicySubjectReviewSpec defines specification for PodSecurityPolicySubjectReview
+type PodSecurityPolicySubjectReviewSpec struct {
+	// PodSpec is the PodSpec to check.
```

Comment:
Indicate that if the `ServiceAccountName` is empty, it will not be defaulted.  If it is non-empty, it will be checked.